docs/user/guides/administrators.rst

```diff
@@ -12,6 +12,7 @@ have a look at our :doc:`/tutorial/index`.
 .. toctree::
    :maxdepth: 1
 
+   Manage Custom Domains <custom-domains>
```

Comment:
We're not worried about the location of this article for now, as we will be adding sub-topics later.